src/test/resources/KeyrefModuleTest/src/test4.ditamap

```diff
@@ -0,0 +1,15 @@
+    <title class="- topic/title " xtrc="title:1;4:10" xtrf="file:/Users/raducoravu/Downloads/4007/map.ditamap">Home</title>
+    <keydef class="+ map/topicref mapgroup-d/keydef " keys="Tool" processing-role="resource-only" xtrc="keydef:1;5:23" xtrf="file:/Users/raducoravu/Downloads/4007/map.ditamap">
+        <topicmeta class="- map/topicmeta " xtrc="topicmeta:1;6:16" xtrf="file:/Users/raducoravu/Downloads/4007/map.ditamap">
+            <keywords class="- topic/keywords " xtrc="keywords:1;7:17" xtrf="file:/Users/raducoravu/Downloads/4007/map.ditamap">
+                <keyword class="- topic/keyword " xtrc="keyword:1;8:18" xtrf="file:/Users/raducoravu/Downloads/4007/map.ditamap">My Product</keyword>
+            </keywords>
+        </topicmeta>
+    </keydef>
+    <mapref class="+ map/topicref mapgroup-d/mapref " format="ditamap" href="submap.ditamap" keyscope="submap" processing-role="resource-only" scope="peer" xtrc="mapref:1;12:97" xtrf="file:/Users/raducoravu/Downloads/4007/map.ditamap"/>
+    <submap class="+ map/topicref mapgroup-d/topicgroup ditaot-d/submap " dita-ot:orig-class="+ map/topicref mapgroup-d/mapref " dita-ot:orig-format="ditamap" dita-ot:orig-href="submap.ditamap" dita-ot:submap-DITAArchVersion="1.3" dita-ot:submap-cascade="merge" dita-ot:submap-class="- map/map " dita-ot:submap-domains="(topic abbrev-d) (topic delay-d) (map ditavalref-d) (map glossref-d) (topic hazard-d) (topic hi-d) (topic indexing-d) (map mapgroup-d) (topic markup-d xml-d) (topic markup-d) (topic pr-d) (topic relmgmt-d) (topic sw-d) (topic ui-d) (topic ut-d) a(props deliveryTarget)" dita-ot:submap-specializations="@props/deliveryTarget" dita-ot:submap-xtrc="map:1;3:6" dita-ot:submap-xtrf="file:/Users/raducoravu/Downloads/4007/submap.ditamap" keyscope="submap" xtrc="mapref:2;13:52" xtrf="file:/Users/raducoravu/Downloads/4007/map.ditamap"><submap-topicmeta class="+ map/topicmeta ditaot-d/submap-topicmeta "><submap-title class="+ topic/navtitle ditaot-d/submap-title " dita-ot:submap-class="- topic/title " dita-ot:submap-xtrc="title:1;4:10" dita-ot:submap-xtrf="file:/Users/raducoravu/Downloads/4007/submap.ditamap">Submap</submap-title></submap-topicmeta><topicgroup class="+ map/topicref mapgroup-d/topicgroup " keyscope="topics" xtrc="topicgroup:1;5:33" xtrf="file:/Users/raducoravu/Downloads/4007/submap.ditamap">
+        <topicref class="- map/topicref " href="topic1.dita" xtrc="topicref:1;6:35" xtrf="file:/Users/raducoravu/Downloads/4007/submap.ditamap"/>
```

Comment:
Are the absolute `file:/` paths okay here?

Reply:
They do not influence the test, but I just removed the path which was specific to my operating system nonetheless.